app/controllers/users_controller.rb

```diff
@@ -13,6 +14,7 @@ def create
     if new_user.save
       new_user = User.create(user)
       flash[:success] = "Welcome, #{new_user.email}!"
+      session[:user_id] = new_user.id
       redirect_to dashboard_path
     else
       # find way to notify error for p.w. and email seperately 
```

Comment:
Let's make sure we revisit this and add an explicit message for the passwords not matching